FlightStreamDeck.Logics/Actions/GenericGaugeAction.cs

```diff
@@ -272,6 +293,14 @@ private async Task UpdateImage()
                 }
             }
         }
+
+        private string precision
+        {
+            get
+            {
+                return $"F{((settings.ValuePrecision?.Length ?? 0) > 0 ? settings.ValuePrecision : this.defaultSettings.ValuePrecision)}";
+            }
+        }
```

Comment:
Don't need to change this, but C# has a nice syntax for get-only property that you might want to consider:

```
private string Precision 
    => $"F{((settings.ValuePrecision?.Length ?? 0) > 0 ? settings.ValuePrecision : this.defaultSettings.ValuePrecision)}";
```
This shows the intention quite nicely and doesn't waste too much space 😄.